Documentation/ApiOverview/ContentElements/CustomBackendPreview.rst

```diff
@@ -14,6 +14,65 @@ following sections describe how to achieve that.
 A preview renderer is used to facilitate (record) previews in TYPO3. This
 class is responsible for generating the preview and the wrapping.
 
+The default PreviewRenderer is :php:`StandardContentPreviewRenderer` and handles
+the core's built-in CTypes.
+
+Extend the default PreviewRenderer
+==================================
+
+There's two ways of how you can provide previews for your own custom CTypes:
+via PageTS or event listener.
+
+PageTS
+------
+
+..  code-block:: typoscript
+
+    mod.web_layout.tt_content.preview.<CType> = EXT:my_extension/Resources/Private/Templates/Preview/ExampleCType.html
+
+For more details see :ref:`TSconfig Reference <t3tsconfig:pageweblayoutpreview>`
+
+Event listener
+--------------
+
+..  versionadded:: 12.0
+    Since version 12.0 this this technique replaces the former hook
+    :php:`$GLOBALS['TYPO3_CONF_VARS']['SC_OPTIONS']['cms/layout/class.tx_cms_layout.php']['tt_content_drawItem']`
+
+The event :php:`PageContentPreviewRenderingEvent` is being dispatched by the
+:php:`StandardContentPreviewRenderer`. You can listen to it with your own
+event listener.
+
+..  code-block:: yaml
+    :caption: EXT:my_extension/Configuration/Services.yaml
+
+    services:
+      Vendor\MyExtension\EventListener\ExampleCType:
```

Comment:
```suggestion
      MyVendor\MyExtension\EventListener\ExampleCType:
```

We always try to use "MyVendor" to prevent confusion with the directory "vendor" for newbies.